jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.cpp

```diff
@@ -944,53 +1010,7 @@ ecma_date_to_string (ecma_value_t this_arg, /**< this argument */
     prim_value_prop_p = ecma_get_internal_property (obj_p, ECMA_INTERNAL_PROPERTY_PRIMITIVE_NUMBER_VALUE);
```

Comment:
Do we really need the `ecma_op_to_object`? It seems to me that the `else` block is not executed if `this_arg` contains a non-object value.